maintainers/scripts/README.md

```diff
@@ -0,0 +1,22 @@
+# Maintainer scripts
+
+This folder contains various executable scripts for nixpkgs maintainers,
+and supporting data or nixlang files as needed.
+
+What follows is a (very incomplete) overview of available scripts.
+
+
+## Metadata
+
+### `get-maintainer.sh`
+
+`get-maintainer.sh [selector] [value]` returns a JSON object describing
```

Comment:
```suggestion
`get-maintainer.sh [selector] value` returns a JSON object describing
```

`value` is not optional

Reply:
Oh I'm just suggesting to not have the `[]` brackets. That usually indicates optional fields in the CLI, whereas if you don't have them it indicates a required field.